### PR TITLE
Backup page - past events

### PR DESCRIPTION
**Pull request by wschenkel**
Hello @paulodiovani ,

Follow the pull request for issue #15 .

Added items:
- Past events backup page;
- Past events on menu;
- Restructuring assets folder.

one
two

### CODE REVIEW

**Comment by wschenkel** [comment]
Hello @paulodiovani ,

If you have any improvement suggestions, please let me know.

File: index.html [hunk 1]
```diff
@@ -69,12 +69,18 @@ <h1><a href="#intro">Hacktober<span>fest</span>NH</a></h1>
           <!-- <li><a href="#hotels">Hotéis</a></li> -->
           <!-- <li><a href="#gallery">Galeria</a></li> -->
           <li><a href="#supporters">Patrocínio</a></li>
+          <li>
+            <a class="dropdown-toggle" href="javascript:void(0);" data-toggle="dropdown">Edições anteriores</a>
+            <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+              <a class="dropdown-item" href="./past_events/2019/index.html">2019</a>
+            </div>
+          </li>
```
Comment (paulodiovani) [inline index.html:77]: Can we change the dropdown background color?

image

**Review by paulodiovani — APPROVED** [review]
Nice. :)

Buy there are conflicts.

File: index.html (hunk 1 above)
Comment (wschenkel) [reply index.html:77]: Hi @paulodiovani 

The background-color was changed.

IMG_2039

**Comment by wschenkel** [comment]
> Nice. :)
> 
> Buy there are conflicts.

@paulodiovani  The conflicts were solved!

**Review by paulodiovani — APPROVED** [review]

**Pull request merged**